sources/models/src/lib.rs

```diff
@@ -261,6 +261,8 @@ struct KubernetesSettings {
     cluster_domain: DNSDomain,
     node_ip: IpAddr,
     pod_infra_container_image: SingleLineString,
+    // Generated in K8s 1.26 only
```

Comment:
Nit: should this say `1.26 and up`?

Reply:
We expect this issue to go away in 1.27 when `aws-iam-authenticator` and the control plane side logic gets updated. So we'd no longer need the `--hostname-override` as this is a temporary measure.